tensorboard/plugins/scalar/tf_scalar_dashboard/tf-scalar-card.html

```diff
@@ -252,6 +253,14 @@
           },
         },
 
+        // A function called to fetch the scalars data from the backend.
+        // Should receive a {tag, run, experiment} object and return
+        // a promise resolving with the fetched scalars. The default
+        // implementation of this function executes:
+        // this.requestManager.request(
+        //      this.getDataLoadUrl(tag, run, experiment})
```

Comment:
Should add opening squiggly brackets :)
`// this.getDataLoadUrl({tag, run, experiment})`

Reply:
Done.